src/uwtools/config.py

```diff
@@ -79,6 +83,9 @@ def replace_templates(self):
         replacing Jinja2 templates as necessary. This is a placeholder
         until more details are fleshed out in work scheduled for PI6.'''
 
+    def config_dump(self):
+        ''' This method will print configure contents'''
+        print(json.dumps(self.data))
```

Comment:
I'm not entirely sure we need this method, but am not completely against it. The reason I'm thinking that is because I was hoping that if we printed a Config object, we'd see its data contents and maybe not have to call a method.

A test case that I was thinking for that scenario might be something like:

```
assert print(cfg) == print(json.dumps(expected))
```